fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/AbstractPhysicalJoin.java

```diff
@@ -183,4 +184,17 @@ public Optional<MarkJoinSlotReference> getMarkJoinSlotReference() {
     public void addFilterConjuncts(Collection<Expression> conjuncts) {
         filterConjuncts.addAll(conjuncts);
     }
+
+    @Override
+    public JSONObject toJson() {
```

Comment:
AbstractPhysicalJoin.java extends PhysicalBinary.
This override will make different behavior between PhysicalJoin and Other Plan

Reply:
Because abstractPhysicalJoin can be different with abstractLogicalJoin, so her can have some different implementation